r/R/rzeppelin/R/zzz.R

```diff
@@ -0,0 +1,9 @@
+typeMap <- list()
```

Comment:
Could you please explain, as this looks different from https://github.com/cran/rscala/blob/master/R/zzz.R